src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -272,6 +272,21 @@ public void logGrantedPrivileges(String privilege, TransportRequest request, Tas
         }
     }
 
+    @Override
+    public void logIndexEvent(String privilege, TransportRequest request, Task task) {
+        if(!checkTransportFilter(AuditCategory.INDEX_EVENT, privilege, getUser(), request)) {
+            return;
+        }
+        // log only cluster admin action
+        if (!privilege.startsWith("indices:admin/")) {
```

Comment:
Why are we ignoring non cluster admin?

Reply:
We are interested only in Index actions https://opendistro.github.io/for-elasticsearch-docs/docs/security/access-control/permissions/#indices

Reply:
Index actions also include indices:data/* and indices:monitor/* options

Reply:
indices:data involves document indexing and indices:monitor involves monitoring requests. 
We are only interested in requests affecting an index like index is created, deleted, settings updated, close, open, update mappings so on.

Reply:
Also document data access is already covered under compliance

Reply:
Thanks for the information.